ingest/ingest_pipeline.py

```diff
@@ -176,14 +176,32 @@ def initialize_file_connection(self, file_type, file_path):
                 File object.
         """
         file_connections = {"cell_metadata": CellMetadata, "cluster": Clusters}
-
-        return file_connections.get(file_type)(
-            file_path,
-            self.study_id,
-            self.study_file_id,
-            tracer=self.tracer,
-            **self.kwargs,
-        )
+        try:
+            return file_connections.get(file_type)(
+                file_path,
+                self.study_id,
+                self.study_file_id,
+                tracer=self.tracer,
+                **self.kwargs,
+            )
+        except ValueError as v:
+            # Caution: recording errorTypes in this manner can clobber other collected errors.
+            # ValueErrors during file connection indicate file cannot be processed
+            # this logging approach should not lose collected file validation information
+            if str(v).startswith("could not convert"):
+                config.get_metric_properties().update(
+                    {"errorTypes": ["content:type:not-numeric"], "numErrortypes": 1}
+                )
+            elif str(v).startswith("Unable to parse"):
+                config.get_metric_properties().update(
+                    {"errorTypes": ["format:cap:unique"], "numErrortypes": 1}
+                )
+            else:
+                config.get_metric_properties().update(
+                    {"errorTypes": ["parse:unhandled"], "numErrortypes": 1}
```

Comment:
Blocker: Mixpanel property names should be lower camel case to avoid downstream analytics UX bugs:

```suggestion
                    {"errorTypes": ["content:type:not-numeric"], "numErrorTypes": 1}
                )
            elif str(v).startswith("Unable to parse"):
                config.get_metric_properties().update(
                    {"errorTypes": ["format:cap:unique"], "numErrorTypes": 1}
                )
            else:
                config.get_metric_properties().update(
                    {"errorTypes": ["parse:unhandled"], "numErrorTypes": 1}
```

Two separate non-blocking issues:

First, the casing above reveals a bug in [`set_mixpanel_num`](https://github.com/broadinstitute/scp-ingest-pipeline/blob/334b09a117110237823c0c0fb55f8895c92a0c09/ingest/config.py#L63).  Using `my_string.capitalize()` capitalizes to first letter and _lowercases all subsequent letters_, which I think neither of us realized.  So, what is currently

`num_prop = "num" + prop.capitalize()`

should -- in order to achieve camel casing -- be something like

`num_prop = "num" + prop[0].upper() + prop[1:]`

This is worth ticketing for this sprint or next if you opt to not fix it as part of this PR.

Second, why omit the standard `message` item in the issue list?  If there's a trivial way to access that data in these three log points, it'd be worthwhile to enhance debuggability via Mixpanel -- e.g.  to be able to query Mixpanel on what the user reported in a support ticket, or to trivial learn the value in the user's data that failed validation.  

If it's not trivial to include `message` here, it's likely not worthwhile.

Reply:
Hm, I had forgotten about `set_mixpanel_num` so the specification of numErrorTypes here is not necessary.
updated `set_mixpanel_num` in `45230a2`

Will create ticket for restoring error messages to mixpanel logging.